### PR TITLE
Fully cover commands with unit tests 3

### PR DESCRIPTION
**Pull request by okozlovlux**
- created UT for basic funcs in RequestFromHMI
  - added UT for next commands:
     UiScrollableMessageResponse,
     GenericResponse,
     ListFilesRequest

Related to [APPLINK-28073](https://adc.luxoft.com/jira/browse/APPLINK-28073)

### CODE REVIEW

**Comment by okozlovlux** [comment]
@OHerasym @dtrunov @LevchenkoS @wolfylambova22 @pvvasilev @Kozoriz @AGritsevich  please review

File: src/components/application_manager/test/commands/mobile/list_files_request_test.cc [hunk 1]
```diff
@@ -102,20 +127,28 @@ TEST_F(ListFilesRequestTest, Run_TooManyHmiNone_UNSUCCESS) {
...
 
-  ON_CALL(*app, GetAvailableDiskSpace()).WillByDefault(Return(0));
+TEST_F(ListFilesRequestTest, Run_EmptyListFiles_SUCCESS) {
+  SharedPtr<ListFilesRequest> command(CreateCommand<ListFilesRequest>());
 
-  am::AppFilesMap files_map;
-  ON_CALL(*app, getAppFiles()).WillByDefault(ReturnRef(files_map));
+  SetExpectCallsForRunSuccess();
 
+  const uint32_t apps_cnt = 0;
```
Comment (LevchenkoS) [inline src/components/application_manager/test/commands/mobile/list_files_request_test.cc]: @okozlovlux, name is not clear.

Comment (okozlovlux) [inline src/components/application_manager/test/commands/mobile/list_files_request_test.cc]: @LevchenkoS fixed d128a7b

**Comment by okozlovlux** [comment]
@wolfylambova22 please review

**Pull request merged**